studio/schemas/documents/salaryAndBenefits.ts

```diff
@@ -21,7 +22,13 @@ const salaryAndBenefits = defineType({
     defineField({
       name: "showSalaryCalculator",
       title: "Show Salary Calculator",
-      description: "Should the salary calculator be visible on the page?",
+      description: MultiLineDescription({
```

Comment:
Why do we need the method MultiLineDescription? Can you add image how this will look?

Reply:
The point was to allow a description with line breaking. This was not possible with just a string literal, but the description property also allows React components.